src/test/skript/tests/regressions/4663-formatted numbers.sk

```diff
@@ -0,0 +1,17 @@
+test "formatted numbers function":
```

Comment:
Make sure to test invalid numbers too! NaN, infinity, null values, etc.
Floating point would also be good to show.